src/actions/channelsActions.js

```diff
@@ -94,7 +105,7 @@ export const depositChannel = (
                             totalDeposit,
```

Comment:
This seems to be bad, youre adding the total_deposit value to itself.

Reply:
Changed the total_deposit for channel_deposit.